src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,42 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void checkSize() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
Collection의 크기를 검증하는 경우에는 다음처럼도 작성할 수 있다는 점 참고차 말씀드립니다.
```suggestion
        assertThat(numbers).hasSize(3);
```